src/resources/js/events-admin.js

```diff
@@ -619,10 +629,18 @@ jQuery( function( $ ) {
 		}
 	}
 
+	window.wp.hooks.addAction( 'tec.events.admin.linked_posts.add_post', 'tec', ( post_type, template_section, new_section ) => {
+		if ( 'tribe_venue' !== post_type ) {
+			return;
+		}
+
+		new_section.find( '#EventCountry' ).trigger( 'change' );
+	} );
+
 	//show state/province input based on first option in countries list, or based on user input of country
-	$( 'body' ).on( 'change', '#EventCountry', function () {
+	$( document ).on( 'change', '[id="EventCountry"]', function () {
```

Comment:
Searching by attribute is less efficient than searching by ID

Reply:
Yeah but it breaks apparently, cause it only applies to the first one in the page. Cause IDs are supposed to be unique one per page.